static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -1,34 +1,50 @@
 # Updating pipelines
 
-If you need to rename or relocate the file paths and/or dependencies of a
-specific DVC-file, then you can 
+If you need to rename or relocate the file paths and/or dependencies and
+outputs of a specific DVC-file, then you can
 
 - use `dvc move` to edit file paths of outputs or dependencies.
-- `dvc run` to make edits in the files and/or dependencies and `dvc commit` to
-  save changes.
+- `dvc run` to make edits in the files and/or dependencies and outputs then
+  `dvc commit` to save changes to cache.
 
-If your source file is a data file, then use `dvc move`:
+If you want to relocate or rename data files and/or directories, dependencies
+and outputs use `dvc move`. `dvc move` is not permitted for stages that are
+not data sources.
+
+```dvc
+$ dvc move output/models/variables/risk_dict.json output/models/risk_dict.json
+?[31mError?[39m: failed to move 'output/models/variables/risk_dict.json' -> 'output/models/risk_dict.json'
```

Comment:
Is this output from Windows? I looks like a bug to me. @efiop do you know if it's expected?

Reply:
Yes, it is from an user but I think its not the latest version and I should change it.